test/integration/skaffold_test.go

```diff
@@ -51,32 +50,51 @@ func TestSkaffold(t *testing.T) {
 	}
 	defer os.Remove(tf.Name())
 
-	// start minikube cluster
-	args := append([]string{"start", "-p", profile, "--memory=2200"}, StartArgs()...)
-	rr, err := Run(t, exec.CommandContext(ctx, Target(), args...))
+	rr, err := Run(t, exec.CommandContext(ctx, tf.Name(), "version"))
+	if err != nil {
+		t.Fatalf("error running skaffold version: %v\n%s", err, rr.Output())
+	}
+	t.Logf("skaffold version: %s", rr.Stdout.Bytes())
+
+	args := append([]string{"start", "-p", profile, "--memory=2600"}, StartArgs()...)
+	rr, err = Run(t, exec.CommandContext(ctx, Target(), args...))
 	if err != nil {
 		t.Fatalf("starting minikube: %v\n%s", err, rr.Output())
 	}
 
 	// make sure minikube binary is in path so that skaffold can access it
 	abs, err := filepath.Abs(Target())
-	// copy minikube binary to minikube
-	if err := mcnutils.CopyFile(Target(), filepath.Join(filepath.Dir(abs), "minikube")); err != nil {
-		t.Fatalf("error copying to minikube")
-	}
 	if err != nil {
-		t.Fatalf("absolute path to minikube binary: %v", err)
+		t.Fatalf("unable to determine abs path: %v", err)
 	}
+
+	oldPath := os.Getenv("PATH")
 	os.Setenv("PATH", fmt.Sprintf("%s:%s", filepath.Dir(abs), os.Getenv("PATH")))
+	defer func() {
+		os.Setenv("PATH", oldPath)
+		t.Logf("PATH is now %s", os.Getenv("PATH"))
+	}()
+
+	t.Logf("PATH is now %s", os.Getenv("PATH"))
+
+	if filepath.Base(Target()) != "minikube" {
```

Comment:
Should we set the PATH env var after this part?

Reply:
Done.